infrastructure/template.yaml

```diff
@@ -428,6 +440,11 @@ Resources:
         BearerTokenName: !Ref BearerTokenName
         CommonStackName: !Ref CommonStackName
         SsmParametersFunction: !Ref SsmParametersFunction
+        CheckHmrcEventBus: !Ref CheckHmrcEventBus
+        CheckHmrcEventBusSource: !FindInMap [Audit, Source, !Ref Environment]
+        AuditEventResponseReceived:
+          !FindInMap [Audit, EventName, ResponseReceived]
+        AuditEventRequestSent: !FindInMap [Audit, EventName, RequestSent]
```

Comment:
same as above the event should be REQUEST_SENT and RESPONSE_RECEIVED and the you'll have to need the thing that brings the event structure to create the actual name that would be sent off to TxMA see https://github.com/govuk-one-login/ipv-cri-check-hmrc-api/blob/77d189b157c7d31ea7d54bbbbe4021e925de20f5/step-functions/audit_event_shaper.asl.json#L81